servant-swagger-ui.cabal

```diff
@@ -69,12 +68,18 @@ extra-source-files:
     jensoleg-dist/swagger.json
 
     index.html.tmpl
-    swagger-ui-dist-3.4.0/favicon-16x16.png
-    swagger-ui-dist-3.4.0/favicon-32x32.png
-    swagger-ui-dist-3.4.0/swagger-ui-bundle.js
-    swagger-ui-dist-3.4.0/swagger-ui-standalone-preset.js
-    swagger-ui-dist-3.4.0/swagger-ui.css
-    swagger-ui-dist-3.4.0/swagger-ui.js
+    swagger-ui-dist-3.9.1/favicon-16x16.png
+    swagger-ui-dist-3.9.1/favicon-32x32.png
+    swagger-ui-dist-3.9.1/index.html
+    swagger-ui-dist-3.9.1/oauth2-redirect.html
+    swagger-ui-dist-3.9.1/swagger-ui-bundle.js
+    swagger-ui-dist-3.9.1/swagger-ui-bundle.js.map
+    swagger-ui-dist-3.9.1/swagger-ui-standalone-preset.js
+    swagger-ui-dist-3.9.1/swagger-ui-standalone-preset.js.map
+    swagger-ui-dist-3.9.1/swagger-ui.css
+    swagger-ui-dist-3.9.1/swagger-ui.css.map
```

Comment:
Please remove map files, without them you won't need to tweak GHC memory (which I won't accept anyway, that should be configured externally, e.g. In `stack.yaml`)

Reply:
Makes sense. I did this, and also made another small mention in the README instructions about it.